### DISCUSSION TITLE
How to know the specific address that uct_iface_get_device_address obtained

### DISCUSSION THREAD

**Discussion by w7658610** [openucx/ucx#7728] [Q&A] [answered]
like uct_iface_get_device_address,uct_ep_get_address,Can we get the specific address?Thanks

**Comment by yosefe** [comment, marked as answer]
the values returned from uct_iface_get_device_address and uct_ep_get_address are device-independent and their internal structure is not exported in UCT API, by design.

**Reply by w7658610** [reply]
thanks
